src/services/inventoryService.ts

```
@@ -163,248 +256,143 @@ export const addItem = async (
+                    addMiscItems(inventory, miscItemChanges);
                     await inventory.save();
                     return {
                         InventoryChanges: {
-                            Consumables: consumablesChanges
+                            MiscItems: miscItemChanges
+                        }
+                    };
+                }
+                break;
+            case "Restoratives": // Codex Scanner, Remote Observer, Starburst
+                const inventory = await getInventory(accountId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)